com.unity.render-pipelines.core/ShaderLibrary/EntityLighting.hlsl

```diff
@@ -121,6 +121,14 @@ float3 SampleSH9(float4 SHCoefficients[7], float3 N)
     return res;
 }
 
+void GetCornetteShanksPhaseFunction(out float zh[3], float anisotropy)
```

Comment:
Worth looking a bit at the produced code, but often I saw arrays like this compiling into weird stuff. 
Safest bet is to use float3 here though it'd mean slightly changing the code. 

Alternatively, it's fine to keep as is if we check the disasm